pyproject.toml

```diff
@@ -114,9 +115,10 @@ select = [
     "I",
     # Pyupgrade
     "UP",
+    "NPY201",
```

Comment:
Does this enable ruff to explicitely cover for numpy2 deprecations?

Reply:
Yes indeed this is what I used to quickly fix all the numpy2 issues.

This actually enabled me to simply run `ruff check --fix` and that fixed all numpy 2.0 issues automatically. This rule could actually be removed now, since it is mostly just useful during the migration.